lib/easypost/util.rb

```diff
@@ -37,10 +39,12 @@ module EasyPost::Util
   BY_TYPE = {
     'Address' => EasyPost::Address,
     'ApiKey' => EasyPost::ApiKey,
+    'BankAccount' => EasyPost::PaymentMethod,
```

Comment:
There is no such thing as a `PaymentMethod` object. We should not include these here

Reply:
See above